app/src/main/java/com/github/se/signify/ui/screens/profile/FriendsListScreen.kt

```diff
@@ -59,7 +60,8 @@ val currentUserId = FirebaseAuth.getInstance().currentUser?.email?.split("@")?.g
 @Composable
 fun FriendsListScreen(
     navigationActions: NavigationActions,
-    userViewModel: UserViewModel = viewModel(factory = UserViewModel.Factory)
+    userRepository: UserRepository,
+    userViewModel: UserViewModel = viewModel(factory = UserViewModel.factory(userRepository))
```

Comment:
The userViewModel could be a val outside the parameter list of `FriendsListScreen()` to keep coherence with other screen implementation.

Reply:
I had originally done that. Turns out some weird tests in `FriendsListScreenTest` are passing over a `ViewModel` they have a reference to into the screen to test the viewmodel's logic. I don't think these tests should even exist, but left looking into it for a later task.

I should've mentioned this in my description. Good point.